delta/plugins/archive/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/ArchiveDownloadSpec.scala

```diff
@@ -57,6 +57,8 @@ class ArchiveDownloadSpec
     with EitherValuable
     with CatsRunContext
     with CatsIOValues
+    with IOValues
```

Comment:
This plugin as been migrated, we should not see things related to Monix

Reply:
Ah, this is because `AclSimpleCheck` hasn't been migrated yet